specs/multicloud/Huawei_Cloud_File_Block.md

```diff
@@ -0,0 +1,931 @@
+                "Kind": {
+                    "BoolValue": false
+                }
+            },
+            "Links": {
+                "Kind": {
+                    "StructValue": {
+                        "fields": {
+                            "href": {
+                                "Kind": {
+                                    "StringValue": "<Links>"
+                                }
+                            },
+                            "rel": {
+                                "Kind": {
+                                    "StringValue": "self"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "ProjectID": {
+                "Kind": {
+                    "StringValue": "<ProjectID>"
+                }
+            },
+            "ShareID": {
+                "Kind": {
+                    "StringValue": "cf149d37-09f6-45eb-8c73-8713f0abffeb"
+                }
+            },
+            "ShareNetworkID": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "ShareType": {
+                "Kind": {
+                    "StringValue": "default"
+                }
+            },
+            "SnapshotID": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "VolumeType": {
+                "Kind": {
+                    "StringValue": "default"
+                }
+            }
+        }
+    }
+}
+```
+**Delete File share**
+
+DELETE http://192.168.20.162:8089/v1/94b280022d0c4401bcf3b0ea85870519/file/shares/5f87b9bc90f7e10001fa260c
+
+Response:
+
+```
+{}
+```
+
+## Dependencies
+
+The support for Huawei cloud SFS and EVS will use the GOLANG SDK provided. Here is the reference:
+
+### SFS
+[https://github.com/huaweicloud/golangsdk](https://github.com/huaweicloud/golangsdk)
+
+### EVS
+[https://github.com/huaweicloud/huaweicloud-sdk-go-v3](https://github.com/huaweicloud/huaweicloud-sdk-go-v3)
+
+## Dashboard changes: 
+SODA Dashboard need to be updated to accomodate the new Vendor, Huawei Cloud, in the support for File and Block Service
+
+## References:
+[https://github.com/sodafoundation/design-specs/blob/master/specs/multicloud/Block_Storage_Service.md](https://github.com/sodafoundation/design-specs/blob/master/specs/multicloud/Block_Storage_Service.md)
+
+[https://github.com/sodafoundation/design-specs/blob/master/specs/multicloud/File_Storage_Service.md](https://github.com/sodafoundation/design-specs/blob/master/specs/multicloud/File_Storage_Service.md)
+
+[https://support.huaweicloud.com/en-us/productdesc-evs/en-us_topic_0014580741.html](https://support.huaweicloud.com/en-us/productdesc-evs/en-us_topic_0014580741.html)
+
+[https://support.huaweicloud.com/intl/en-us/api-sfs/sfs_02_0002.html](https://support.huaweicloud.com/intl/en-us/api-sfs/sfs_02_0002.html)
+
```

Comment:
in the file multicloudMatrix.png, does this mean that the service is supported for all storage types on a given vendor ?
specifically, is object migration supported for all file, block and object storage of Huawei cloud ?

Reply:
Didn't get this, As per my understanding, Object Migration will be specific to Object storage. Did you mean, *Migration* or Say *Lifecycle* supported for all file, block and object storage?